libp2p/src/tutorials/hole_punching.rs

```diff
@@ -122,16 +123,17 @@
 //!
 //! ``` bash
 //! ## Inside the rust-libp2p repository.
-//! cargo build --example client -p libp2p-dcutr
+//! cd ./examples/relay-server
+//! cargo build
 //! ```
 //!
-//! You can find the binary at `target/debug/examples/client`. In case you built it locally, copy
+//! You can find the binary at `target/debug/dcutr`. In case you built it locally, copy
 //! it to your listening client machine.
 //!
 //! On the listening client machine:
 //!
 //! ``` bash
-//! RUST_LOG=info ./client --secret-key-seed 1 --mode listen --relay-address /ip4/$RELAY_SERVER_IP/tcp/4001/p2p/12D3KooWDpJ7As7BWAwRMfu1VU2WCqNjvq387JEYKDBj4kx6nXTN
+//! RUST_LOG=info ./dcutr --secret-key-seed 1 --mode listen --relay-address /ip4/$RELAY_SERVER_IP/tcp/4001/p2p/12D3KooWDpJ7As7BWAwRMfu1VU2WCqNjvq387JEYKDBj4kx6nXTN
```

Comment:
I can't comment on it but I think line 158 also needs to be updated! :)

Reply:
Ok, I updated line 158. Thank a lot